docs/user/faq.rst

```diff
@@ -103,19 +109,20 @@ run into any issues, please let us know.
 Does Falcon support asyncio?
 ------------------------------
 
-Due to the limitations of WSGI, Falcon is unable to support ``asyncio`` at this
-time. However, we are exploring alternatives to WSGI (such
-as `ASGI <https://github.com/django/asgiref/blob/master/specs/asgi.rst>`_)
-that will allow us to support asyncio natively in the future.
+Starting with version 3.0, the `ASGI <https://asgi.readthedocs.io/en/latest/>`_
+flavor of Falcon now proudly supports :any:`asyncio`!
+Use the :class:`falcon.asgi.App` class to create an async application.
```

Comment:
I think we could mention uvicorn here

Reply:
That wouldn't hurt. :+1: 
Do you maybe have a specific suggestion in your mind?
I would ideally just link to the [Installation](https://falcon.readthedocs.io/en/latest/user/install.html) article, but it only discusses WSGI servers atm...

Reply:
> Use the :class:`falcon.asgi.App` class to create a async applications and serve them using an ASGI compatible library, like uvicorn.

Something along these lines. Mainly since we mention a couple of wsgi servers below

Reply:
Thanks, :+1: to that.
I'll try to add that Installation chapter anyway (to provide further pointers), and otherwise follow your suggestion.